typescript/sdk/src/middleware/account/InterchainAccountDeployer.ts

```diff
@@ -30,14 +30,26 @@ export class InterchainAccountDeployer extends ProxiedRouterDeployer<
     });
   }
 
-  async constructorArgs(_: string, config: RouterConfig): Promise<[string]> {
+  routerContractName<K extends keyof InterchainAccountFactories>(
```

Comment:
why have this take in config if its unused?

Reply:
It's overriding the abstract so keeping the same sig